src/QualificationRunner.Core/Domain/ComparisonTimeProfilePlot.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using OSPSuite.Core.Qualification;
+
+namespace QualificationRunner.Core.Domain
+{
+   public class ComparisonTimeProfilePlot : IReferencingSimulations

```

Comment:
I am defining the minimal structure corresponding to the qualification plan so that I can retrieve the information